tests/e2e/scenarios/test_reborn_webui_v2_product_auth_api.py

```
@@ -0,0 +1,178 @@
+                f"{reborn_v2_server}{path}",
+                json={},
+                timeout=15,
+            )
+            assert response.status_code == 401, (path, response.text)
+
+
+async def test_reborn_v2_product_auth_manual_token_setup_redacts_submit_error_served(
+    reborn_v2_server,
+):
+    headers = reborn_bearer_headers()
+    raw_token = "ghp_served_product_auth_secret_must_not_echo"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "raw_token"

(S105)